spec/fortnox/api/validators/base_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+require 'fortnox/api/validators/base'
+require 'fortnox/api/validators/validator_examples'
+
+describe Fortnox::API::Validator::Base do
+  subject{ Class.new{ include Fortnox::API::Validator::Base }.new }
+
+  it_behaves_like 'validators' do
+    before do
+      subject.using_validations do
```

Comment:
You mean the `subject.using_validations` part? Then the test fails with:
```
Failure/Error: raise ArgumentError, "No validator given" unless @validator
```
The `validators` shared example assumes that the validatos is initialized.